config/deploy/production.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-remote_user = Net::SSH::Config.for('umasstransit.rodeo')[:user] || ENV['USER']
-server 'umasstransit.rodeo',
-       roles: %w[app db web],
-       user: remote_user
-set :tmp_dir, "/tmp/#{remote_user}"
```

Comment:
This was a workaround for different users trying to use the same git wrapper. Capistrano now makes a new one every t.ime